vizro-core/src/vizro/static/css/figures.css

```diff
@@ -1,50 +1,97 @@
 .card-kpi {
-  border-left: 4px solid var(--text-secondary);
+  border-left: 4px solid var(--bs-secondary);
+  height: 100%;
+  padding: 1rem;
+}
+
+/* We currently ship this CSS file with the package, so the font definitions need to live here at the moment.
+The CSS is taken from https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined and the corresponding font file
+downloaded to fonts/material-symbols-outlined.woff2. See:
+https://developers.google.com/fonts/docs/material_symbols#self-hosting_the_font
+https://fonts.google.com/knowledge/using_type/self_hosting_web_fonts
+Note the relative url works with both serve_locally=True and serve_locally=False through the CDN.
+*/
+@font-face {
+  font-family: MaterialSymbolsOutlined;
+  src: url("./fonts/material-symbols-outlined.woff2") format("woff2");
+}
+
+.material-symbols-outlined {
+  direction: ltr;
+  display: inline-block;
+  font-family: MaterialSymbolsOutlined, sans-serif;
+  font-feature-settings: "liga";
+  font-size: 24px;
+  -webkit-font-smoothing: antialiased;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: normal;
+  line-height: 1;
+  text-transform: none;
+  white-space: nowrap;
+  word-wrap: normal;
+}
+
+.card-kpi .material-symbols-outlined {
+  margin: auto 0;
+}
+
+/* This should be replaced, when the bootstrap theme is finalized. Currently our font tokens don't match with the
+ bootstrap font tokens, leading to deviations which we need to overwrite. */
+.card-kpi-title {
+  font-size: 1.5rem;
+  line-height: 1;
+  margin: 0;
+}
+
+.card-kpi .card-header,
+.card-kpi .card-footer,
+.card-kpi .card-body {
+  background: none;
+  border: none;
+  margin: 0;
+  padding: 0;
 }
 
 .card-kpi .card-header {
   display: flex;
   flex-direction: row;
-  gap: 8px;
-}
-
-.card-kpi .card-header .material-symbols-outlined {
-  line-height: 30px;
+  gap: 0.5rem;
 }
 
 .card-kpi .card-body {
   align-items: center;
-  color: var(--text-secondary);
+  color: var(--bs-secondary);
```

Comment:
Important for future reference: In case we do add new components, colors need to be taken from the bootstrap variables, otherwise the new components are not compatible with other bootstrap themes.

Reply:
Shall we document this in our wiki, and potentially also in our code?

Reply:
There is a ticket on creating a guide on an "intro to css" where we could add this in 👍 

So i wouldn't add it to this PR for now. All PRs changing CSS would currently go through @nadijagraca, @pruthvip15  and/or me anyway :)